admin_2017.rb

```diff
@@ -20,110 +20,138 @@ def source_dir
 ####################################
 # GEMS
 ####################################
-
 gem 'bootstrap_form'
 gem 'kaminari'
 gem 'haml', '~> 5.0'
+gem 'devise'
+gem 'carrierwave'
```

Comment:
Was the plan to still use carrierwave, or just the hyrez thing? I can't remember where we landed on that.

Reply:
carrierwave can still be used to upload the image itself.  hyrez is given the original image's url that carrierwave uploaded.

Of course, carrierwave is not as vital as it was and we could just store an image url in db, so dunno if the plan is to not use it anymore.